docs/APP.md

```diff
@@ -108,13 +108,19 @@ const frontendOfApp = new AragonApp(new providers.WindowMessage(window.parent))
 
 Get an array of the accounts the user currently controls over time.
 
-Returns **[Observable](https://rxjs-dev.firebaseapp.com/api/index/class/Observable)**: An multi-emission observable that emits an array of account addresses every time a change is detected.
+Returns **[Observable](https://rxjs-dev.firebaseapp.com/api/index/class/Observable)**: A multi-emission observable that emits an array of account addresses every time a change is detected.
```

Comment:
I snuck in this little grammar change :) But let me know if you want me to remove